sktime/forecasting/model_evaluation/tests/test_evaluate.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Testt evaluate."""
```

Comment:
Maybe a bit more descriptive? "Tests for model evaluation module", etc

Reply:
Now it reads:
```
"""Tests for model evaluation module.

In particular, function `evaluate`, that performs time series
cross-validation, is tested with various configurations for correct output.
"""
```

Reply:
ok, thanks!